src/Traits/GqlIterator.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Softonic\GraphQL\Traits;
+
+trait GqlIterator
```

Comment:
In this case I'd transform it better in a class implementing the `\Iterator` interface

Reply:
Why? Could you please explain how you'd solve the same issue with a class? Inheritance?